internal/usage/usage.go

```diff
@@ -15,66 +15,70 @@
+	SSL                      = "Flag that indicates if TLS is enabled for the sync source database."
+	OplogSSL                 = "Flag indicating whether this oplog store only accepts connections encrypted using TLS."
+	SyncSource               = "mongod instance from which you retrieve backup data."
+	ExcludedNamespace        = "List of database names and collection names to omit from the backup."
+	IncludedNamespace        = "List of database names and collection names to include in the backup."
+	TeamUsername             = "List of usernames to add to the new team."
+	DBUsername               = "Username for authenticating to MongoDB."
+	TeamName                 = "Name of the team."
+	UserID                   = "Unique identifier of the user."
+	LDAPHostname             = "Hostname or IP address of the LDAP server."
+	LDAPPort                 = "Port to which the LDAP server listens to for client connections."
+	Hostname                 = "The fully qualified domain name (FQDN) of the target node that should receive the authentication attempt."
+	BindUsername             = "The user distinguished name (DN) that Atlas uses to connect to the LDAP server."
+	BindPassword             = "Password used to authenticate the bindUsername."
+	CaCertificate            = "CA certificate used to verify the identify of the LDAP server."
+	AuthzQueryTemplate       = "An LDAP query template that Atlas executes to obtain the LDAP groups to which the authenticated user belongs."
+	MappingMatch             = "An ECMAScript-formatted regular expression (regex) to match against a provided username."
+	MappingLdapQuery         = "An LDAP query formatting template that inserts the authentication name matched by the match regex into an LDAP query URI encoded respecting RFC4515 and RFC4516."
+	MappingSubstitution      = "An LDAP distinguished name (DN) formatting template that converts the authentication name matched by the match regex into a LDAP DN."
+	AuthenticationEnabled    = "Specifies whether user authentication with LDAP is enabled."
+	AuthorizationEnabled     = "Specifies whether user authorization with LDAP is enabled."
+	TeamID                   = "Unique identification of the team."
+	Password                 = "User's password."
+	Country                  = "The ISO 3166-1 alpha two-letter country code of the user's country of residence."
+	Mobile                   = "The user's mobile or cell phone number."
+	Period                   = "Duration in ISO 8601 that specifies how far back in the past to retrieve measurements."
+	Roles                    = "User's roles and the databases or collections on which the roles apply."
+	RolesExtended            = `User's roles and the databases or collections on which the roles apply.
+The roles format is roleName[@dbName[.collection]].
+roleName can either be a builtin role or a custom role.
+dbName and collection are optional and only needed for builtin roles.`
```

Comment:
should this be
```suggestion
dbName and collection are optional and only allowed for builtin roles.`
```
I guess docs can decide